### DISCUSSION TITLE
New Ledger Reports GET_LEDGER_DETAIL_VIEW_DATA & GET_LEDGER_SUMMARY_VIEW_DATA -  Adjustment Documentation

### DISCUSSION THREAD

**Discussion by nrogers805** [amzn/selling-partner-api-models#594] [Ideas]
I'm looking for documentation for the reason code given for Adjustments in the new Inventory Ledger reports.

I have information for these codes for the original adjustment report but the new ledger report has introduced new codes, example new code is "G"

**Comment by nrogers805** [comment]
Why has this not been answered ?

**Comment by sungolivia** [comment]
this is a duplicate of https://github.com/amzn/selling-partner-api-docs/issues/2986 please refer to that issue!
